### PR TITLE
Added likes page and implemented api calls

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
# Added likes page and implemented API calls
## Preview: [lucent-video-lib](https://deploy-preview-5--ym-video-lib-dev.netlify.app/)
-----> Click on the categories on the homepage to visit the video listing page in the preview.
-----> On clicking the watch now button, it will be redirected to the video page and also adds the data to the history page at the same time.

### CODE REVIEW

File: src/backend/db/videos.js [hunk 1]
```diff
@@ -118,6 +118,6 @@ export const videos = [
       "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
     creator: "Soham Shah",
     watchLater: false,
-    history: false,
+    liked: false,
```
Comment (tejas-muthya37) [inline src/backend/db/videos.js]: Not sure why you are using this attribute, you can add a video to the liked and always run a find to know if it is liked or not, basically whether if it's present in the liked array or not.

File: src/components/cards/video-card/VideoCard.jsx [hunk 2]
```diff
@@ -32,10 +33,12 @@ const VideoCard = ({
...
     <>
       <div className="video-card-container card-pos-rel">
-        {history && (
-          <MdDelete
-            className="video-card-icon cursor color-red"
-            onClick={removeFromHistory}
+        {!watchLater && (
+          <AiFillLike
+            className={`video-card-icon cursor ${
+              liked ? "color-green" : "color-black"
```
Comment (tejas-muthya37) [inline src/components/cards/video-card/VideoCard.jsx]: Feel like the class names here can be a little more generic like liked-video-card

**Review by sri-rishi — APPROVED** [review]
Code and UI looks good, just one suggestion your `like-function.js` didn't contain any other function except than api calls, So I think giving it name like `likepageApiCalls` would make some sense

File: src/utility/history-functions.js [hunk 3]
```diff
@@ -49,14 +44,14 @@ const removeFromHistory = async (id, historyDispatch) => {
     });
 
     if (response.status === 200) {
-      const historyTrue = [...response.data.history].map((historyData) => ({
-        ...historyData,
-        history: true,
-      }));
+      // const historyTrue = [...response.data.history].map((historyData) => ({
```
Comment (tejas-muthya37) [inline src/utility/history-functions.js]: Can remove these comments if not required

**Review by tejas-muthya37 — APPROVED** [review]
Code and UI look good.

**Pull request merged**